ipatests/test_xmlrpc/test_user_plugin.py

```diff
@@ -53,6 +53,7 @@
 
 invaliduser1 = u'+tuser1'
 invaliduser2 = u''.join(['a' for n in range(256)])
+invaliduser3 = u'1234'
```

Comment:
test_create_with_numeric_only_username should use invaliduser3 instead of hardcoding u'1234'